content/en/synthetics/guide/browser_tests_without_chrome_extension.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Datadog Browser tests without the Chrome extenstion
```

Comment:
So we're actually still using the Chrome extension for recording, the difference is that users need to install that extension in a different way